### PR TITLE
Created video page

### PR DESCRIPTION
**Pull request by AnnaMinasian**
develop

## GitHub Board

**Issue link**

[#271 create video page ](https://github.com/ita-social-projects/dokazovi-fe/issues/271)

## Summary of changes

- [x]  added video page
- [x] added description editor

### CODE REVIEW

File: src/lib/components/Editor/GeneralEditor.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { setIsDone } from '../../../modules/postCreation/store/postCreationSlice
 import { PostTypeEnum } from '../../types';
 
 export interface IQuillEditorProps {
-  type: 'ARTICLE' | 'DOPYS';
+  type: 'ARTICLE' | 'DOPYS' | 'VIDEO';
```
Comment (SergeOlabin) [inline src/lib/components/Editor/GeneralEditor.tsx:13]:
```suggestion
  type: PostTypeEnum,
```

**Review by SergeOlabin — APPROVED** [review]
pls use enum in the comment place

File: src/lib/components/Editor/GeneralEditor.tsx (hunk 1 above)
Comment (AnnaMinasian) [reply src/lib/components/Editor/GeneralEditor.tsx]: fixed

**Comment by SergeOlabin** [comment]
pls relove conflicts

**Pull request merged**